vador/src/main/java/com/salesforce/vador/execution/strategies/AnnotationProcessorBase.kt

```diff
@@ -0,0 +1,52 @@
+package com.salesforce.vador.execution.strategies
+
+import com.salesforce.vador.annotation.ValidateWith
+import com.salesforce.vador.lift.liftToEtr
+import com.salesforce.vador.types.Validator
+import com.salesforce.vador.types.ValidatorAnnotation1
+import com.salesforce.vador.types.ValidatorEtr
+import io.vavr.Tuple2
+import java.lang.reflect.InvocationTargetException
+import org.apache.commons.lang3.reflect.FieldUtils
+
+object AnnotationProcessorBase {
```

Comment:
`AnnotationProcessorBase` - Pls remove `Base` from the name as this is not ancestor for any class